javascript/javascript3/week2/Movies/movies_array.js

```diff
@@ -0,0 +1,39 @@
+//1. Create an array of bad movies
+const apiUrl = `https://tinyurl.com/y9zck5xx`;
+// Bad movies rated for <=4
+async function badMoviesArray() {
+    //a.fetching the movies data
+    fetch(apiUrl)
+        .then((response) => response.json())
+        .then((movies) => {
+            //b.filtering bad movies accrotding to ratings
+            const badMovies = movies
+                .filter((movie) => movie.rating <= 4)
+                .map((movie) => movie);
+            console.log(badMovies);
+        })
+        .catch((error) => {
+            //b.Throw an error if above condition failed
```

Comment:
Throw comment is unnecessary. 
Imo, comments should be avoided. Better to write declaritve code